execution-engine/contracts/hdac-system/pop-install/src/lib.rs

```diff
@@ -185,6 +185,15 @@ pub extern "C" fn call() {
         runtime::revert(Error::NotMatchedTotalBondAndDelegate);
     }
 
+    // Insert genesis blocktime to last distribute time
+    // "ld_{last_distribute_time}"
+    let genesis_blocktime: u64 = runtime::get_blocktime().into();
```

Comment:
I am curious would it be okay to use genesis_config::block_time as last_distribute_time.

Reply:
Is there a better way to get birth date of the network life?

Reply:
We are supporting exporting and importing the states of this contract. And we can restart from the specific state. My question is is your intention to use the birth date of the network as last_distribute_time.